docs/install/lotus/troubleshooting.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Troubleshooting Lotus installation
+description: If you're having issues installing Lotus on your system, try running through some of these troubleshooting steps.
+---
+
+# Troubleshooting
+
+If you're having trouble installing Lotus, try running through some of these troubleshooting steps. If you're still having trouble, feel free to reach out on the `#fil-help` channel in the Filecoin Slack.
+
+## Update your operating system
+
+This sounds simple, but a common reason users aren't able to run Filecoin is because their operating system isn't up to date.
```

Comment:
This is not very good advise. What are the operating systems supported or known to work? Does this mean updating the installed packages or making sure you are on the latest release of the OS? I would be very surprised if someone followed the install instructions and finds that Lotus does not work.

Also, it should say run Lotus (not run Filecoin) ?

I guess this can be fixed later, but maybe move down as it is less likely to help than the others.

Reply:
Removing this for now.